helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -390,8 +390,213 @@ compatibility with the old config group name: "otelK8sClusterReceiver".
 {{- define "splunk-otel-collector.o11yInfraMonEventsEnabled" -}}
 {{- $clusterReceiver := fromYaml (include "splunk-otel-collector.clusterReceiver" .) }}
 {{- if eq (toString $clusterReceiver.k8sEventsEnabled) "<nil>" }}
-{{- .Values.splunkObservability.infrastructureMonitoringEventsEnabled }}
+{{- .Values.splunkObservability.infrastructureMonitoringObjectsEnabled }}
```

Comment:
"objects enabled" is more ambiguous than "events enabled" so how about something like "k8s object reporting enabled" for all applicable vars?

Reply:
Below @dmitryax wrote `infrastructureMonitoringEventsEnabled  is the name of some functionality in Splunk Observability IM`, so I think I shouldn't have changed it at all